tiledb/cf/xarray_engine/_common.py

```diff
@@ -0,0 +1,3 @@
+_VARIABLE_UNLIMITED_DIMS_PREFIX = "__xr_variable_unlimited_dimensions."
+_VARIABLE_ATTR_NAME_PREFIX = "__xr_variable_attribute_name."
```

Comment:
Should we leave these as xarray specific key names (`__xr_*`) or try to make them more TileDB or Dataspace specific?